sheetstorm/urls.py

```diff
@@ -19,4 +19,5 @@
     url(r"^", include("users.urls")),
     url(r"^managers/", include("managers.urls")),
     url(r"^", include("employees.urls")),
+    url(r"^select2/", include("django_select2.urls")),
```

Comment:
According to docs (https://django-select2.readthedocs.io/en/latest/django_select2.html#module-django_select2.urls), it is only need when "you use any ‘Model’ fields". I have no idea what the mean by that - that when we use it for `ModelForms`? Anyway, it would be great if you could check if this is really needed (I am very careful when it comes to adding new urls from 3rd party libs).

Reply:
it seems to work without it - good catch! :+1: